lacework/utils.go

```diff
@@ -17,11 +28,32 @@ func castAndTransformStringArray(iArray []interface{}, f func(string) string) []
 	return a
 }
 
-// turn a string array into an instance array
-func castStringArrayToInterface(strs []string) []interface{} {
+// turn a string slice into an instance slice
+func castStringSliceToInterface(strs []string) []interface{} {
```

Comment:
I don't see any usages of this func

Reply:
oh right, back in the days, I thought this could be a good way to also transform/validate fields,
but to be honest, we haven’t used it :sweat_smile: I am happy to remove it if it is too complex.